### PR TITLE
Dart support

### PR DESCRIPTION
**Pull request by fundthmcalculus**
This has vaccine demo working on my machine, but it is the initial pass of dart support.

### CODE REVIEW

**Comment by fundthmcalculus** [comment]
Ignore the golang failure, that's a style (`gofmt`) issue which will be addressed in a different PR.

File: dart/example/vaccine_example.dart [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+import 'package:trinsic_dart/src/proto/services/universal-wallet/v1/universal-wallet.pbgrpc.dart';
+import 'package:trinsic_dart/src/proto/services/verifiable-credentials/v1/verifiable-credentials.pbgrpc.dart';
+import 'package:trinsic_dart/src/trinsic_util.dart';
+import 'package:trinsic_dart/trinsic.dart';
+
+String baseDataPath() {
+  return path.join(path.current, "..", "devops", "testdata");
+}
+
+String vaccineCertUnsignedPath() {
```
Comment (geel9) [inline dart/example/vaccine_example.dart:14]: The vaccine example needs to be updated in line with https://github.com/trinsic-id/sdk/issues/659

Given that this still needs to happen for Go and Java, though (and the fact that I'm the done that's been making the changes) I wouldn't mind approving this PR now and doing it myself with the others -- good chance to play around with Dart a bit.

File: python/samples/vaccine_demo.py [hunk 2]
```diff
@@ -4,176 +4,167 @@
 
 from trinsic.account_service import AccountService
```
Comment (geel9) [inline python/samples/vaccine_demo.py]: I think you need to merge main into your branch

File: dart/lib/src/dart_base.dart [hunk 3]
```diff
@@ -1,6 +1,255 @@
...
+import 'package:trinsic_dart/src/proto/sdk/options/v1/options.pb.dart';
+import 'package:trinsic_dart/src/proto/services/account/v1/account.pbgrpc.dart';
+import 'package:trinsic_dart/src/proto/services/provider/v1/provider.pbgrpc.dart';
+import 'package:trinsic_dart/src/proto/services/trust-registry/v1/trust-registry.pbgrpc.dart';
+import 'package:trinsic_dart/src/proto/services/universal-wallet/v1/universal-wallet.pbgrpc.dart';
+import 'package:trinsic_dart/src/proto/services/verifiable-credentials/templates/v1/templates.pbgrpc.dart';
+import 'package:trinsic_dart/src/proto/services/verifiable-credentials/v1/verifiable-credentials.pbgrpc.dart';
+import 'package:trinsic_dart/src/service_base.dart';
 
-/// Checks if you are awesome. Spoiler: you are.
```
Comment (geel9) [inline dart/lib/src/dart_base.dart:3]: This seems to be an erroneous delete -- I'm definitely still awesome.

This is a blocking issue.

**Review by geel9 — CHANGES_REQUESTED** [review]

File: dart/example/vaccine_example.dart (hunk 1 above)
Comment (fundthmcalculus) [reply dart/example/vaccine_example.dart:14]: I vote you do it. That way, everything is done consistently.

File: python/samples/vaccine_demo.py (hunk 2 above)
Comment (fundthmcalculus) [reply python/samples/vaccine_demo.py]: Yup lol. Will do.

File: dart/lib/src/dart_base.dart (hunk 3 above)
Comment (fundthmcalculus) [reply dart/lib/src/dart_base.dart:3]: I got you. 😀

**Review by geel9 — APPROVED** [review]

**Pull request merged**